src/components/Router.js

```diff
@@ -38,30 +38,32 @@ const pageWrapStyle = { display: 'flex', flexDirection: 'column', flex: '1 0 aut
 
 const initNavPaths = () => {
   Nav.clearPaths()
-  LandingPage.addNavPaths()
-  WorkspaceList.addNavPaths()
-  WorkflowView.addNavPaths()
-  StyleGuide.addNavPaths()
-  ImportData.addNavPaths()
-  ImportTool.addNavPaths()
-  PrivacyPolicy.addNavPaths()
-  Dashboard.addNavPaths()
-  Data.addNavPaths()
-  Notebooks.addNavPaths()
-  JobHistory.addNavPaths()
-  SubmissionDetails.addNavPaths()
-  Tools.addNavPaths()
-  NotebookLauncher.addNavPaths()
-  Profile.addNavPaths()
-  Groups.addNavPaths()
-  Group.addNavPaths()
-  TerminalLauncher.addNavPaths()
-  TermsOfService.addNavPaths()
-  Code.addNavPaths()
-  DataExplorer.addNavPaths()
-  Datasets.addNavPaths()
-  Showcase.addNavPaths()
-  Projects.addNavPaths()
+  _.forEach(Nav.defPath, [
+    ...LandingPage.navPaths,
```

Comment:
I'd love it if these didn't all have to be spread. Can we tuck a flatten in here?